tests/tests/custom_authority_tests.cpp

```diff
@@ -1257,6 +1259,571 @@ BOOST_AUTO_TEST_CASE(custom_auths) { try {
+         //            "argument": [
+         //              39,
+         //              [
+         //                {
+         //                  "member_index": 1,
+         //                  "restriction_type": 0,
+         //                  "argument": [
+         //                    8,
+         //                    "1.3.4"
+         //                  ],
+         //                  "extensions": []
+         //                }
+         //              ]
+         //            ],
+         //            "extensions": []
+         //          }
+         //        ],
+         //        [
+         //          {
+         //            "member_index": 2,
+         //            "restriction_type": 10,
+         //            "argument": [
+         //              39,
+         //              [
+         //                {
+         //                  "member_index": 1,
+         //                  "restriction_type": 0,
+         //                  "argument": [
+         //                    8,
+         //                    "1.3.5"
+         //                  ],
+         //                  "extensions": []
+         //                }
+         //              ]
+         //            ],
+         //            "extensions": []
+         //          }
+         //        ],
+         //        [
+         //          {
+         //            "member_index": 2,
+         //            "restriction_type": 10,
+         //            "argument": [*
+         //              39,
+         //              [
+         //                {
+         //                  "member_index": 1,
+         //                  "restriction_type": 0,
+         //                  "argument": [
+         //                    8,
+         //                    "1.3.5"
+         //                  ],
+         //                  "extensions": []
+         //                }
+         //              ]
+         //            ],
+         //            "extensions": []
+         //          }
+         //        ]
+         //      ]
+         //    ],
+         //    "extensions": []
+         //  },
+         //  {
+         //    "member_index": 3,
+         //    "restriction_type": 10,
+         //    "argument": [
+         //      39,
+         //      [
+         //        {
+         //          "member_index": 1,
+         //          "restriction_type": 0,
+         //          "argument": [
+         //            8,
+         //            "1.3.2"
+         //          ],
+         //          "extensions": []
+         //        }
+         //      ]
+         //    ],
+         //    "extensions": []
+         //  }
+         //]
+
+         // Broadcast the authorization
+         trx.clear();
+         trx.operations = {authorize_limit_orders};
+         sign(trx, alice_private_key);
+         PUSH_TX(db, trx);
+
+
+         // Authorize the cancellation of orders
```

Comment:
Hm, this shows a limitation of the concept. It is not possible to restrict the cancelling to orders in specific markets...

Reply:
With the current and only operation to cancel a limit order, this is true.

An alternate cancel operation which does not require a limit order ID but instead has parameters for sell and/or receive asset types (such as to cancel all open orders where BTS is being sold) could then be used by this restricted trade case for a more tightly restricted ability to cancel.